uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/leave-type-wizard.component.js

```diff
@@ -134,18 +143,22 @@ define([
     function initValidatorsForField (field) {
       $scope.$watch(function () {
         return field.value;
-      }, function (value) {
-        delete field.error;
+      }, function (value, oldValue) {
```

Comment:
Please rename value as `newValue`. It will be easier to read.

Reply:
done